Formula/vectorscan.rb

```diff
@@ -0,0 +1,39 @@
+class Vectorscan < Formula
+  desc "High-performance regular expression matching library"
+  homepage "https://github.com/VectorCamp/vectorscan"
+  url "https://github.com/VectorCamp/vectorscan/archive/refs/tags/vectorscan/5.4.7.tar.gz"
+  sha256 "cd70c2a7bf632b5374083a450019703605520c10c5614a4c12011c99ab8435dd"
+  license "BSD-3-Clause"
+
+  depends_on "boost" => :build
+  depends_on "cmake" => :build
+  depends_on "pkg-config" => :build
+  depends_on "ragel" => :build
+  depends_on arch: :arm64
```

Comment:
Doesn't this build on x86_64 too? Or are they removing all the non-ARM code eventually?

Reply:
That is a good point:  it is indeed supported.  However, building fails on account of a dependence on a non-macos mktemp in one of its architecture-dependent cmake files.  I've logged [this upstream bug](https://github.com/VectorCamp/vectorscan/issues/111).

Given that, contrary to my original assertion, hyperscan and vectorscan are not mutually exclusive, I will add a `conflicts_with` directive.

Reply:
> Given that, contrary to my original assertion, hyperscan and vectorscan are not mutually exclusive, I will add a `conflicts_with` directive.

Auditing now fails with an additional
```
  * Formula hyperscan should also have a conflict declared with vectorscan
```
Should I PR a suitable change for hyperscan?